### PR TITLE
PIM-5665 : Add doc about remove action on products rules

### PR DESCRIPTION
**Pull request by momoss**

### CODE REVIEW

File: cookbook/rule/general_information_on_rule_format.rst [hunk 1]
```diff
@@ -275,10 +275,33 @@ Two parameters are required, two other are optional.
...
+Remove
+______
+
+This action removes values to a multiselect, a category or a collection.
+
+Two parameters are required, two other are optional.
+ - field: attribute code.
+ - locale: local code for which value is assigned (optional).
+ - scope: channel code for which value is assigned (optional).
+ - items: attribute value
```
Comment (grena) [inline cookbook/rule/general_information_on_rule_format.rst]: attribute values to remove

**Comment by grena** [comment]
Looks good, but could be nice to have a good english proof reader :dancer: 
GTM

**Comment by jjanvier** [comment]
GTM if items changed by values

**Comment by momoss** [comment]
Fix "items" in rule "add" documentation, it's the same attribute for "add" and "remove"

**Pull request merged**